.gitlab/rc_kubernetes_deploy.yml

```
@@ -0,0 +1,29 @@
+---
+# internal_kubernetes_deploy stage
+# Contains jobs to trigger a pipeline in our k8s-datadog-agent-ops repo to deploy release candidate build
+
+rc_kubernetes_deploy:
+  stage: internal_kubernetes_deploy
+  rules:
+  - if: $RC_K8S_DEPLOYMENTS == "true"
+    when: always
+  - !reference [.on_deploy_a7]
+  needs:
+  - job: docker_trigger_internal
+    artifacts: false
+  - job: docker_trigger_cluster_agent_internal
+    artifacts: false
+  - job: k8s-e2e-main # Currently only require container Argo workflow
+    artifacts: false
+    optional: true
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  variables:
+    OPTION_AUTOMATIC_ROLLOUT: "true"
+    OPTION_PRE_SCRIPT: "patch-cluster-images-operator.sh env=all-staging ${CI_COMMIT_REF_SLUG}-jmx ${CI_COMMIT_REF_SLUG}"
+    SKIP_PLAN_CHECK: "true"
+    EXPLICIT_WORKFLOWS: "//workflows:deploy_rc.agents_rc"
+  script:
+    - source /root/.bashrc
+    - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_pipelines_scheduler_token --with-decryption --query "Parameter.Value" --out text)
```

```suggestion
    - set +x
    - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_pipelines_scheduler_token --with-decryption --query "Parameter.Value" --out text)
```
It's often done like this to reduce risk of leak

---

Good catch, changed. I also quickly checked and it would be good to add that to multiple other places too, but I think it should be in a different PR.

---

I already created a ticket for this on agent-platform side (APL-2384)